src/components/Executions/Tables/test/NodeExecutionsTable.test.tsx

```diff
@@ -113,19 +121,121 @@ describe('NodeExecutionsTable', () => {
                     value={requestConfig}
                 >
                     <ExecutionContext.Provider value={executionContext}>
-                        <TestTable />
+                        <NodeExecutionDetailsContextProvider
+                            workflowId={mockWorkflowId}
+                        >
+                            <TestTable />
+                        </NodeExecutionDetailsContextProvider>
                     </ExecutionContext.Provider>
                 </NodeExecutionsRequestConfigContext.Provider>
             </QueryClientProvider>
         );
 
+    describe('when rendering the DetailsPanel', () => {
```

Comment:
This test is moved as is with addition of fetchWorkflow mock implementation from the bottom of the file, as I wasn't able to fully clear jest.spyOn() effect, which was messing up with this test return.
added:
```
 fetchWorkflow.mockImplementation(() =>
                Promise.resolve(fixture.workflows.top)
            );
```